src/renderer/screens/lend/Account/AccountBodyHeader.js

```diff
@@ -0,0 +1,179 @@
+// @flow
+import React, { useCallback, useMemo } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Trans } from "react-i18next";
+import styled from "styled-components";
+import type { TokenAccount, Account } from "@ledgerhq/live-common/lib/types";
+
+import { makeCompoundSummaryForAccount } from "@ledgerhq/live-common/lib/compound/logic";
+import { getAccountUnit, getAccountCurrency } from "@ledgerhq/live-common/lib/account";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/lib/currencies";
+
+import { openModal } from "~/renderer/actions/modals";
+import Text from "~/renderer/components/Text";
+import Button from "~/renderer/components/Button";
+import Box, { Card } from "~/renderer/components/Box";
+import LinkWithExternalIcon from "~/renderer/components/LinkWithExternalIcon";
+import Header from "./Header";
+import Row from "./Row";
+
+import moment from "moment";
+import { useDiscreetMode } from "~/renderer/components/Discreet";
+import { localeSelector } from "~/renderer/reducers/settings";
+
+type Props = {
+  account: TokenAccount,
+  parentAccount: Account,
+};
+
+const Wrapper = styled(Box).attrs(() => ({
+  p: 3,
+  mt: 24,
+  mb: 6,
+}))`
+  border: 1px dashed ${p => p.theme.colors.palette.text.shade20};
+  border-radius: 4px;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const Loans = ({ account, parentAccount }: Props) => {
+  const dispatch = useDispatch();
+  const locale = useSelector(localeSelector);
+  const unit = getAccountUnit(account);
+  const discreet = useDiscreetMode();
+  const currency = getAccountCurrency(account);
+
+  const formatConfig = useMemo(
+    () => ({
+      disableRounding: false,
+      alwaysShowSign: false,
+      showCode: true,
+      discreet,
+      locale,
+    }),
+    [discreet, locale],
+  );
+
+  const summary = makeCompoundSummaryForAccount(account, parentAccount);
+  const lendingDisabled = false;
+
+  const onLending = useCallback(() => {
+    dispatch(
+      openModal("MODAL_LEND_HIGH_FEES", {
+        nextModal: ["MODAL_LEND_ENABLE_INFO", { account, parentAccount, currency }],
+      }),
+    );
+  }, [dispatch, account, parentAccount, currency]);
+
+  const formattedClosedLoans = useMemo(
```

Comment:
the usage of memo here is not helpful because the `makeCompoundSummaryForAccount` will always give a new reference.

I think the pattern involved here is not really great and that we should directly do `summary.closed.map(closed => <RowSomething closed={closed} />`. Making a component (per row) will really help optimisation. inside that row component, you do the actual work => there you can do the memoization of each field (date, etc..)